Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+pipeline {
+  agent { label "android" }
+
+  stages {
+    stage("File Setup") {
+      steps {
+        sh "mv .jenkins/ci-local.properties local.properties"
+        sh "docker create -t -i -v $WORKSPACE:/opt/project-android --name jd-container simplymadeapps/docker-android:1.0.0"
+        sh "docker start jd-container"
+      }
+    }
+    
+    //We are running as a circleci user and not as root so we have to add sudo to everything
+    
+    stage("Tests") {
+      steps {
+        sh "docker exec jd-container sudo ./gradlew clean"
+        sh "docker exec jd-container sudo ./gradlew createOfflineTestCoverageReport"
+      }
+   	}
+    
+    stage("Coverage") {
+      steps {
+        sh "docker exec jd-container sudo ./gradlew jacocoTestCoverageVerification -x test"
```

Comment:
Purposely fail a build for test coverage reasons so Brandon can see it fail.

Reply:
https://ci.simpleinout.app/blue/organizations/jenkins/PreferenceHelper/detail/PR-2/4/pipeline/